pkg/cmd/install.go

```diff
@@ -799,10 +800,28 @@ func installCommand() *cli.Command {
 					return err // we want the user to see the error message without a prefix
 				}
 			}
+
+			if !isAirgap && license != nil {
```

Comment:
nit: it would cleaner to have this entire block be part of the `maybePromptForAppUpdate` function

Reply:
updated